cristin-import/src/main/java/no/unit/nva/cristin/mapper/CristinObject.java

```diff
@@ -157,4 +152,8 @@ public void hardcodePublicationOwner(String publicationsOwner) {
     public boolean isPeerReviewed() {
         return nonNull(yearReported);
     }
+
+    public List<CristinAssociatedUri> getCristinAssociatedUris() {
+        return nonNull(cristinAssociatedUris) ? cristinAssociatedUris : List.of();
```

Comment:
`Collections.emptyList()` shows intention while `List.of()` looks like we forgot something